modules/chroma/testcontainers/chroma/__init__.py

```diff
@@ -0,0 +1,89 @@
+from typing import TYPE_CHECKING
+
+import chromadb
+from requests import ConnectionError, get
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.utils import raise_for_deprecated_parameter
+from testcontainers.core.waiting_utils import wait_container_is_ready
+
+if TYPE_CHECKING:
+    from requests import Response
+
+
+class ChromaContainer(DockerContainer):
+    """
+    The example below spins up a ChromaDB container, performs a healthcheck and creates a collection.
+    The method :code:`get_client` can be used to create a client for the Chroma Python Client.
+
+    Example:
+
+        .. doctest::
+
+            >>> import io
+            >>> from testcontainers.chroma import ChromaContainer
+
+            >>> with ChromaContainer() as chroma:
+            ...   client = chroma.get_client()
+            ...   client.heartbeat()
+            ...   client.get_or_create_collection("test")
+    """
+
+    def __init__(
+        self,
+        image: str = "chromadb/chroma:latest",
+        port: int = 8000,
+        **kwargs,
+    ) -> None:
+        """
+        Args:
+            image: Docker image to use for the MinIO container.
+            port: Port to expose on the container.
+            access_key: Access key for client connections.
+            secret_key: Secret key for client connections.
+        """
+        raise_for_deprecated_parameter(kwargs, "port_to_expose", "port")
+        super().__init__(image, **kwargs)
+        self.port = port
+
+        self.with_exposed_ports(self.port)
+        # self.with_command(f"server /data --address :{self.port}")
+
+    def get_client(self, **kwargs) -> chromadb.ClientAPI:
+        """Returns a Chroma HttpClient to connect to the container.
+
+        Returns:
+            HttpClient: Chroma HttpClient according to
+                   https://docs.trychroma.com/reference/Client#httpclient
+        """
+        host_ip = self.get_container_host_ip()
+        exposed_port = self.get_exposed_port(self.port)
+        return chromadb.HttpClient(host=host_ip, port=int(exposed_port))
```

Comment:
Can we please avoid coupling to the chroma API?

Reply:
Done